### ISSUE TITLE
Why models perform worse in the 60% identity split than in the 30% identity split on LBA tasks?

### ISSUE THREAD

**Issue by Gloria-LIU** [drorlab/atom3d#57]
Hi,
Thank you for the amazing work!
I am curious about the results in table 8. Why most models (other than GNN) perform dramatically worse in the 60% identity split than in the 30% identity split? Intuitively, the task with 60% split should be easier and achieve better performance as there is more similarity between protein sequences.

**Comment by smiles724** [comment]
> Hi, Thank you for the amazing work! I am curious about the results in table 8. Why most models (other than GNN) perform dramatically worse in the 60% identity split than in the 30% identity split? Intuitively, the task with 60% split should be easier and achieve better performance as there is more similarity between protein sequences.

I agree with your theoretical guess. However, not only atom3d but also some other following studies show the similiar phenomenon. For instance, the table below is copied from [Multi-Scale Representation Learning on Proteins](https://papers.nips.cc/paper/2021/hash/d494020ff8ec181ef98ed97ac3f25453-Abstract.html)
![image](https://user-images.githubusercontent.com/39299863/184093254-cedb404b-6a9d-4749-b9e1-1f2a098001a7.png)
